excercises/panic_R.py

```diff
@@ -0,0 +1,16 @@
+phrase = "Don't panic!"
+plist = list(phrase)
+print(phrase)
+print(plist)
+
+telist = ["D", "'", "n", "i", "c", "!"]
```

Comment:
To ofc jest dobrze, ale może w ramach kontynuowania challengu spróbowałbyś wybrać te litery używając indeksu i slice notation:
num[1],
num[-1] - dla ostatniego elementu,
num[2:5] - dla 3 elementów o indeksach od 2 do 4

Ref:
https://railsware.com/blog/python-for-machine-learning-indexing-and-slicing-for-lists-tuples-strings-and-other-sequential-types/